tests/p4rt/test_p4rt_acl.py

```diff
@@ -241,26 +235,6 @@ def test_AclRulesAddUpdateDelPass(self, dvs, testlog):
         assert status == True
         util.verify_attr(fvs, attr_list)
 
-        # query application state database for ACL tables
```

Comment:
Since its deleted, where is the check for APPL_STATE_DB entries?

Reply:
P4Orch will no longer write to APPL_STATE_DB, so there is no APPL_STATE_DB entries for P4RT table.
(P4Orch writes to APPL_DB instead, which is no change at the moment as APPL_DB is already written when orchagent pops the request entries.)

But P4Orch will still send the response to the Redis channel, which is checked and verified in util.verify_response().
And the test also checks the APPL_DB entries. For now it doesn't mean much. But in the future when we upstream the ZMQ changes, P4Orch will not write to APPL_DB in pop; it will write to APPL_DB in response path instead. At that time the P4RT table in APPL_DB will represent the state (not the intent).